### PR TITLE
storage navigation refactor

### LINKED ISSUE #2091: [Storage] Main Navigation Refactor
- Move API Keys into main navigation
- Handle the Subscription screen in main navigation (hide based on soft-launch flag)
- Leave current Settings route in the code, but hide Nav element (for now)

### PR DESCRIPTION
**Pull request by tanmoyAtb**
closes #2091

We now have Api Keys and Billing nav items in place.
Settings are kept hidden.

### CODE REVIEW

**Comment by tanmoyAtb** [comment]
The settings tests are failing. Going to add the updates.

**Comment by asnaith** [comment]
Working well and looks good for the most part but I'm a little bit undecided on whether this spacing makes sense now: 

Screen Shot 2022-06-20 at 3 15 03 PM

**Review by Tbaut — APPROVED** [review]
The code looks good. I have no strong opinion regarding the spacing, I'm undecided.

File: packages/storage-ui/src/Components/Layouts/AppNav.tsx [hunk 1]
```diff
@@ -13,17 +13,20 @@ import {
   formatBytes,
   ChainsafeLogo,
   FolderSvg,
-  SettingSvg,
+  // SettingSvg,
```
Comment (juans-chainsafe) [inline packages/storage-ui/src/Components/Layouts/AppNav.tsx]: this can be removed I think

**Review by juans-chainsafe — COMMENTED** [review]
Same as Tbaut, I'm not so sure about the spacing, maybe @serenaho can help? 

Regarding the changes, looks good! waiting for the fix of the tests to approve it

**Comment by serenaho** [comment]
Included some options [here](https://www.figma.com/file/alES1NMSWR0ismBtjfYo9N/ChainSafe-Storage?node-id=503%3A1028) on Figma!

Options **A, B,** and **C** are pretty straightforward but options **D** and **E** have Docs and Billing separated as buttons because they don't quite fit under the category as a menu item like the others. 

Happy to discuss this during the planning meeting tmr :)

**Comment by Tbaut** [comment]
This could be for another ticket but:
- I think the docs being an external link should indeed be separate
- I think we prob don't need headers or lines, there's already enough text
- Billing is a wrong name, I think we use "Subscription" in Files

My suggestion would be all together but Docs at the bottom of the nav bar

**Review by juans-chainsafe — APPROVED** [review]
Subscription test were failing in Files, but was a problem with the API that was fixed, I rerun the tests and now they all passed.

Everything fine now! I guess we leave the spacing improvement for another ticket

**Pull request merged**